selfdrive/ui/qt/offroad/wifiManager.cc

```diff
@@ -39,9 +39,10 @@ const QString ipv4config_iface       = "org.freedesktop.NetworkManager.IP4Config
 
 const QString nm_service             = "org.freedesktop.NetworkManager";
 
-const int state_connected = 100;
 const int state_need_auth = 60;
+const int state_connected = 100;
 const int reason_wrong_password = 8;
+const int device_type_wifi = 2;
```

Comment:
is there a header with all these constants we can include?

Reply:
Yeah, this header would be nice to have: https://github.com/NetworkManager/NetworkManager/blob/main/src/libnm-core-public/nm-dbus-interface.h

Reply:
I'll do this in another PR